src/snowflake/cli/_plugins/spcs/services/commands.py

```diff
@@ -221,20 +225,68 @@ def logs(
     num_lines: int = typer.Option(
         500, "--num-lines", help="Number of lines to retrieve."
     ),
+    previous_logs: bool = typer.Option(
+        False, "--previous-logs", "-p", help="Retrieve previous logs", is_flag=True
+    ),
+    since_timestamp: Optional[str] = typer.Option(
+        "", "--since", help="Timestamp to retrieve logs from"
+    ),
+    include_timestamps: bool = typer.Option(
+        False, "--include-timestamps", help="Include timestamps in logs", is_flag=True
+    ),
```

Comment:
What's the purpose of logs if there's no timestamp? Is this a necessary feature? If we don;'t have strong need for it I would suggest to add it later on once we see business/user need.

Reply:
It is necessary, we need timestamps to track logs accurately. Since not all logs have timestamps, tracking becomes difficult. To address this, we introduced the include-timestamps, which adds a timestamp to each log, allowing us to reliably track the last log for the next retrieval cycle.

Reply:
The intention for this flag is as follows, may be we can chose better naming:
1. For every log record K8s adds a timestamp
2. Customers services which emit logs can also have timestamps in some cases.
3. If customer services already emit timestamps k8s timestamps might be redundant.
4. Hence, this flag is to turn on/off the k8s appended timestamps to each log record